### ISSUE TITLE
Common mistake with impact-node create:project

### ISSUE THREAD

**Issue by stahlmanDesign** [roman01la/impact-node#6]
This is just to clarify how to use the commands.

To create a new project, the command `create:project` is not the name of your project, it is the command:

**ERROR `create:mygame`**

**CORRECT `create:project`**

**Comment by roman01la** [comment]
Hi. Is it something you constantly getting into? If so, how do you think we could improve this?

**Comment by roman01la** [comment]
a possible command can be:

```
$ impact-node create:project {project name} --impact={path to impact dir}
```

**Comment by stahlmanDesign** [comment]
I don't think you need to change it, just make it clear in the documentation. But if you do change it, your solution is good.
